lib/core/BlogPost.js

```diff
@@ -56,11 +56,11 @@ class BlogPost extends React.Component {
           </a>
         </div>
       );
-    } else if (post.authorImage) {
```

Comment:
I didn't realize there already exists an option to customize the author thumbnail via `authorImage` because it's undocumented...